BenchmarkVCFs/FindSamplesAndSimpleBenchmark.wdl

```diff
@@ -0,0 +1,146 @@
+version 1.0
+
+import "SimpleBenchmark.wdl" as SimpleBenchmark
+import "../Utilities/WDLs/MatchFingerprints.wdl" as MatchFingerprints
+import "../Utilities/WDLs/CombineTables.wdl" as CombineTables
+
+struct VcfData {
+    File vcf
+    File index
+}
+
+struct BaseVcfData {
+    File vcf
+    File index
+    File eval_intervals
+}
+
+# Searches for matching samples in base_vcfs vs query_vcfs and then runs benchmarking on them
+# Sample names are inferred from the VCFs and collected from the fingerprinting step
+workflow FindSamplesAndSimpleBenchmark {
+    input {
+        Array[File] base_vcfs
+        Array[File] base_vcf_indices
+
+        Array[File] query_vcfs
+        Array[File] query_vcf_indices
+
+        # Reference information
+        File ref_fasta
+        File ref_index
+
+        # Subsetting inputs using intervals
+        Array[File] stratifier_intervals = []
+        Array[String] stratifier_labels = []
+
+        # Evaluation inputs
+        Array[File] evaluation_intervals
+        String score_field = "GQ"
```

Comment:
Is score field usually in the FORMAT field? I'd have guessed that the default here should have been QUAL or VQSLOD or some other INFO level field?

Reply:
I kept the default to GQ since that's the default for the flag in vcfeval itself